docs/community-resources.md

```diff
@@ -0,0 +1,26 @@
+---
+id: community-resources
+title: Community Resources
+sidebar_label: Community Resources
+---
+
+Members of the PayID community have created useful resources for using and learning about PayID.
+
+- [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
```

Comment:
Can we add some subheaders?

Maybe:
- Tutorials
- PayID Server Implementations
- Tools
- Community (Things like the subreddit, and Discord channel).

Reply:
Will do.